fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+    font-family: 'pt_serifbold';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LandingPage/pullRequest?prid=6317355)